docs/releasing-files.md

```diff
@@ -4,42 +4,36 @@
 OpenSAFELY follows the [Five Safes](https://ukdataservice.ac.uk/help/secure-lab/what-is-the-five-safes-framework/) framework for data access to allow safe and efficient use of data. This framework models data access into 5 independent dimensions:
 
 * **Safe projects** - does the project make good use of the data?
-* **Safe people** - are the researchers using the data trustworthy?
-* **Safe data** - is there potential for individuals to be identified in the data?
-* **Safe settings** - are there necessary controls on the data?
+* **Safe people** - are the researchers using the data appropriately trained and aware of their role in data protection?
+* **Safe data** - what is the potential for individuals to be identified in the data?
+* **Safe settings** - are there technical controls on access to the data?
 * **Safe outputs** - is there any residual risk in outputs being released from the environment?
 
-All data in OpenSAFELY is pseudonymised and all projects and researchers in OpenSAFELY go through an approval process before having any access to the dataset. This ensures the “safe data”, “safe projects” and “safe people” elements of the fives safes framework are met. The application of **disclosure controls** and **output-checking** is part of the “safe outputs” dimension.
+All data in OpenSAFELY is pseudonymised and all projects and researchers in OpenSAFELY go through an approval process before having any access to the dataset. This ensures the “safe data”, “safe projects” and “safe people” elements of the fives safes framework are met. The OpenSAFELY platform ensures that handling risk (“safe settings”) is minimsed. The application of **disclosure controls** and **output-checking** is part of the “safe outputs” dimension.
```

Comment:
Typo: minimsed, though I'm not convinced that this sentence is clear.
The technical controls implemented as part of the OpenSAFELY platform ensures that “safe settings” are in place and that risk is minimsed in this respect.

Reply:
Thanks, I've used your suggestion